packages/iTwinUI-react/src/core/Wizard/Stepper.tsx

```diff
@@ -0,0 +1,94 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { useTheme } from '../utils';
+import '@itwin/itwinui-css/css/stepper.css';
+import { StepProperties } from './Wizard';
+import { Step } from './Step';
+
+export type StepperLocalization = {
+  stepsCountLabel: (currentStep: number, totalSteps: number) => string;
+};
+
+export type StepperProps = {
```

Comment:
You should be able to reuse these types inside Wizard to avoid duplication.

Reply:
Changed for here and `WorkflowDiagram`

Reply:
I meant the other way around so that in the future we could remove Wizard without any other changes.

Reply:
All of the Wizard props are now in Stepper, and Wizard now reuses the props defined in Stepper